homeassistant/components/peco/sensor.py

```diff
@@ -0,0 +1,125 @@
+"""Sensor component for PECO outage counter."""
+import asyncio
+from datetime import timedelta
+from types import MappingProxyType
+from typing import Any, Final, cast
+
+from peco import BadJSONError, HttpError
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+PARALLEL_UPDATES: Final = 0
```

Comment:
```suggestion
```
Not sure why it's here but seems it shouldn't

Reply:
It's to comply with platinum requirements.
See [here](https://developers.home-assistant.io/docs/integration_quality_scale_index/?_highlight=PARALLEL_UPDATES#platinum-)

Reply:
There you go. Never noticed it before there.